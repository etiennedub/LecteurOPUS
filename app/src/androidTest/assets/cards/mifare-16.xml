<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">047528D1</List><List index="1">52444984</List><List index="2">DB48E000</List><List index="3">00000003</List><List index="4">00000000</List><List index="5">300B8523</List><List index="6">00000000</List><List index="7">F500FD53</List><List index="8">01C2556C</List><List index="9">56C60506</List><List index="10">19000000</List><List index="11">20287DB7</List><List index="12">01C253B8</List><List index="13">3B860504</List><List index="14">A6008000</List><List index="15">20288904</List></Pages></Mifare>