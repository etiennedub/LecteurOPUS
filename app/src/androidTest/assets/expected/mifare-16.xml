<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="23/09/2016" Time="11:34" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /><List index="1" Date="23/09/2016" Time="07:56" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>